pyuvdata/uvflag.py

```diff
@@ -301,14 +1280,38 @@ def write(self, filename, clobber=False, data_compression='lzf'):
                                            data=self.flag_array,
                                            compression=data_compression)
 
-    def __add__(self, other, inplace=False, axis='time'):
+    def __add__(self, other, inplace=False, axis='time',
+                run_check=True, check_extra=True, run_check_acceptability=True):
         """Add two UVFlag objects together along a given axis.
-        Args:
-            other: second UVFlag object to concatenate with self.
-            inplace: Whether to concatenate to self, or create a new UVFlag object. Default is False.
-            axis: Axis along which to combine UVFlag objects. Default is time.
-        """
 
+        Parameters
+        ----------
+        other : UVFlag
+            object to combine with self.
+        axis: str
+            Axis along which to combine UVFlag objects. Default is time.
```

Comment:
Is it really necessary to specify the axis or can it be figured out from the 2 objects?

Keywords are awkward with double underscore methods because they can only be set by calling the method explicitly using the full name rather than the short cut. I think that's ok for keywords that only rarely need to be modified, but it's a bit weirder for keywords that might be commonly used.

Reply:
@adampbeardsley ?

Reply:
Looks like Adam made an issue to track this, so we can leave it for now.